index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+  <title>Beryl's Portfolio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@100;500&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@100;500&display=swap" rel="stylesheet">
+
+</head>
+<body>
+<main>
+
+  <header>
+    <nav>
+      <ul class="logo" >
+        <li><a href=""> <h3 class="gakii"> Gakii Beryl</h3></a></li>
+        <li><a href=""><img class="menu" src="images/menu.svg" alt="Menu Icon" /></a></li>
+      </ul>
+    </nav>
+  </header>
+  <section class="headline">
+    <div class="about">
+      <h1>Hey There.<br />I'm Gakii </h1>
+      <h2>I'm a software Developer</h2>
+      <p>
```

Comment:
Your design
![image](https://user-images.githubusercontent.com/78251927/226976615-58201297-850c-4beb-9a3d-60c03af41a77.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/226976479-3ae56b15-743c-4cad-b2ec-c53be3fe6346.png)

**[OPTIONAL]**
Kindly update the font of the `<h2>` tag here also to match the given template 💪